Gems/Atom/RHI/Metal/Code/Source/RHI/MetalViewController.mm

```diff
@@ -18,6 +18,18 @@ - (BOOL)prefersStatusBarHidden
     return TRUE;
 }
 
+#if defined(CARBONATED) && defined(AZ_PLATFORM_IOS)
+-(BOOL)prefersHomeIndicatorAutoHidden
```

Comment:
Are you sure this is needed? According to the documentation the default implementation returns NO, the same as here.

Reply:
I believe we want it to return YES here.

Reply:
According to Apple's documentation we'll have the bar at the bottom all the time then. Is it what you see with the test build?